recipes/hiredis/all/conanfile.py

```diff
@@ -15,7 +15,6 @@ class HiredisConan(ConanFile):
     topics = ("hiredis", "redis", "client", "database")
     homepage = "https://github.com/redis/hiredis"
     url = "https://github.com/conan-io/conan-center-index"
-
     settings = "os", "arch", "compiler", "build_type"
```

Comment:
```suggestion
    package_type = "library"
    settings = "os", "arch", "compiler", "build_type"
```

It's a good opportunity to add package_type